src/components/header/NavLogo.tsx

```diff
@@ -3,7 +3,7 @@ import logo from '../../assets/img/logo-icon.svg';
 export const NavLogo = () => {
   return (
     <a href="/">
-      <img src={logo.src} alt="Majáles logo" width={60} height={60} />
+      <img src={logo.src} alt="Majáles logo" width={80} height={80} />
```

Comment:
Radeji bych doporucil embednoud svgcko primo do kodu, nez to tahat pres src, zbytecny request na file.